### DISCUSSION TITLE
Céges logó bizonylat fejlécbe

### DISCUSSION THREAD

**Discussion by AttilaPletyak** [nav-gov-hu/eRECEIPT#10] [Ötletek]
Nemzetközi cégek sokszor kérdezik, hogy a cégük logóját rá lehet-e nyomtatni a bizonylatok fejlécére. Eddig ez nem volt lehetséges. Az új rendeletben szeretném kérni, hogy a jogalkotó térjen ki ennek a kezelésére is.

**Comment by BatariZ** [comment]
Az időközben megjelent rendelet (A nemzetgazdasági miniszter 8/2025. (III. 31.) NGM rendelete az e-pénztárgépek forgalmazásáról, üzemeltetéséről, valamint az e-pénztárgépek és az e-nyugta kiállításának követelményeiről), illetve az aktualizált Fejlesztői dokumentáció tartalmazza a válaszokat a feltett kérdésekre.
